static/app/views/settings/projectGeneralSettings/index.spec.tsx

```diff
@@ -343,7 +342,10 @@ describe('projectGeneralSettings', function () {
           slug: 'new-project',
         },
       });
+    });
 
+    function renderProjectGeneralSettings() {
```

Comment:
pulling this out of beforeEach seems to help with cleanup and how most our tests work